test/starknet/shared/helpers.ts

```diff
@@ -17,3 +32,61 @@ export function bytesToHex(bytes: number[]): string {
   }).join('');
   return '0x' + body;
 }
+
+export async function setup() {
+  const vanillaSpaceFactory = await starknet.getContractFactory(
+    './contracts/starknet/space/space.cairo'
+  );
+  const vanillaVotingStategyFactory = await starknet.getContractFactory(
+    './contracts/starknet/strategies/vanilla_voting_strategy.cairo'
+  );
+  const vanillaAuthenticatorFactory = await starknet.getContractFactory(
+    './contracts/starknet/authenticator/authenticator.cairo'
+  );
+
+  console.log('Deploying auth...');
+  const vanillaAuthenticator = (await vanillaAuthenticatorFactory.deploy()) as StarknetContract;
+  console.log('Deploying strat...');
+  const vanillaVotingStrategy = (await vanillaVotingStategyFactory.deploy()) as StarknetContract;
+  const voting_strategy = BigInt(vanillaVotingStrategy.address);
+  const authenticator = BigInt(vanillaAuthenticator.address);
+  console.log('Deploying space...');
+
+  // This should be declared along with the other const but doing so will make the compiler unhappy as `SplitUin256`
+  // will be undefined for some reason?
```

Comment:
I need some TS gods to help me here... initially had it along with the other consts but it simply didn't work ^^